.github/workflows/push-package-solarwinds.yml

```diff
@@ -0,0 +1,37 @@
+# Copyright (c) 2023 SolarWinds, LLC.
+# All rights reserved.
+
+name: Ruby Gem to Github Package
+
+on:
+  workflow_dispatch:
+    inputs:
+      gem_name:
+        required: true
+        description: 'The name of gem you want to publish (without opentelemetry-instrumentation e.g. aws_sdk)'
+
+jobs:
+  build:
+    name: Build + Publish to Github Package
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Set up Ruby 3.1 and bundle
+        uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: 3.1
+
+      - name: Setup secrets
+        run: |
+          mkdir ~/.gem
+          echo -e "---\n:github: Bearer $GITHUB_SECRET_TOKEN" >> ~/.gem/credentials
```

Comment:
I think this file could be shortened by not setting `env` below and directly using `secrets.GITHUB_TOKEN`

Reply:
Ah ok. We can keep it that way.